code/game/turfs/turf_integrity.dm

```diff
@@ -67,24 +56,6 @@
 	else
 		after_damage(damage_amount, damage_type, damage_flag)
 
-//returns the damage value of the attack after processing the obj's various armor protections
-/turf/run_atom_armor(damage_amount, damage_type, damage_flag = 0, attack_dir, armour_penetration = 0)
```

Comment:
we can dump the whole child proc since its now a duplicate of /atom/proc/run_atom_armor()